subprojects/tooling-api/src/main/java/org/gradle/tooling/TestLauncher.java

```diff
@@ -82,6 +83,26 @@
      */
     TestLauncher withJvmTestMethods(String testClass, Iterable<String> methods);
 
+    /**
+     * Configures test JVM to run in debug mode.
+     * <p>
+     * When called, the forked test JVM is launched with the following argument:
+     * <pre>-agentlib:jdwp=transport=dt_socket,server=n,suspend=n,address=localhost:&lt;port&gt;</pre>
+     * This means the test JVM expects a debugger at the specified port that uses a
+     * <a href="https://docs.oracle.com/javase/1.5.0/docs/guide/jpda/conninv.html#Connectors">socket listening connector</a>.
```

Comment:
I would use Java 6 (the earliest version we support running tests on) or 8.

Reply:
Fixed.